Task_Prototype/index.js

```diff
@@ -1,22 +1,68 @@
-//Write code for the 1st task below this
-
-
-//Write your code above this line
-//Write code for the 2nd task below this
-
-
-//Write your code above this line
-//Write code for the 3rd task below this
```

Comment:
Consider adding a semicolon (;) at the end of the console.log statement to follow best practices.